packages/factory/src/index.ts

```diff
@@ -76,7 +76,7 @@ export const buildPluginFactory = ({
         const context: GlobalContext = getContext({
             start,
             options,
-            bundlerVersion: bundler.version || bundler.VERSION,
+            bundlerVersion: bundler.rspackVersion || bundler.version || bundler.VERSION,
```

Comment:
why are we using 3 tags here?

Reply:
It's to cover:

- `(webpack|esbuild|vite).version`
- `rollup.VERSION`
- `rspack.rspackVersion`

The `bundler` variable is the actual instance of the bundler we're currently using.

Reply:
I see, the variable `bundler` is the library itself

Reply:
Can you add a comment here? 🙏

Reply:
Will do, good idea.